src/wp-includes/rest-api/endpoints/class-wp-rest-sidebars-controller.php

```diff
@@ -234,28 +276,26 @@ protected function do_permissions_check() {
 	 *
 	 * @since 5.8.0
 	 *
-	 * @global array $wp_registered_sidebars The registered sidebars.
-	 *
 	 * @param string|int $id ID of the sidebar.
 	 * @return array|null The discovered sidebar, or null if it is not registered.
 	 */
 	protected function get_sidebar( $id ) {
-		global $wp_registered_sidebars;
-
-		foreach ( (array) $wp_registered_sidebars as $sidebar ) {
-			if ( $sidebar['id'] === $id ) {
-				return $sidebar;
-			}
-		}
+		return wp_get_sidebar( $id );
+	}
 
-		if ( 'wp_inactive_widgets' === $id ) {
-			return array(
-				'id'   => 'wp_inactive_widgets',
-				'name' => __( 'Inactive widgets' ),
-			);
+	/**
+	 * Look for "lost" widgets.
```

Comment:
I'd add `@see retrieve_widgets` to explain what "lost" means

Reply:
Fixed.